backends/platform/sdl/kolibrios/kolibrios.mk

```diff
@@ -0,0 +1,35 @@
+bundle = zip-root
+
+all: scummvm.kos $(EXECUTABLE)
+
+scummvm.kos: $(srcdir)/backends/platform/sdl/kolibrios/wrapper-main.c
+	+$(QUIET)$(CXX) -nostdlib -static  -Wl,-static -Wl,-s -Wl,-T$(KOS32_SDK_DIR)/sources/newlib/app-dynamic.lds -I$(KOS32_SDK_DIR)/sources/newlib/libc/include -Wl,--image-base,0 -x c -o $@ $< -lgcc -lc.dll
+	+$(QUIET)kos32-objcopy $@ -O binary
+
+$(bundle): all
+	$(RM) -rf $(bundle)
+	$(MKDIR) -p $(bundle)/scummvm
+	$(CP) $(DIST_FILES_DOCS) $(bundle)/scummvm
+	$(MKDIR) $(bundle)/scummvm/themes
+	$(CP) $(DIST_FILES_THEMES) $(bundle)/scummvm/themes/
+
+ifdef DIST_FILES_ENGINEDATA
+	$(MKDIR) $(bundle)/scummvm/engine-data
+	$(CP) $(DIST_FILES_ENGINEDATA) $(bundle)/scummvm/engine-data/
+endif
+ifdef DIST_FILES_NETWORKING
+	$(CP) $(DIST_FILES_NETWORKING) $(bundle)/scummvm
+endif
+ifdef DIST_FILES_VKEYBD
+	$(CP) $(DIST_FILES_VKEYBD) $(bundle)/scummvm
+endif
+ifdef DYNAMIC_MODULES
+	$(MKDIR) $(bundle)/scummvm/plugins/
+	$(CP) $(PLUGINS) $(bundle)/scummvm/plugins/
+endif
+	$(CP) scummvm.kos $(bundle)/scummvm/scummvm
+	$(CP) scummvm.dll $(bundle)/scummvm/scummvm.dll
+
+scummvm-zip: $(bundle)
+	$(RM) scummvm_kolibrios.zip
+	cd $(bundle) && zip -r ../scummvm_$(MIYOO_TARGET).zip scummvm
```

Comment:
`MIYOO_TARGET` seems to be a copy/paste error?

Reply:
Done